tests/TestBase.py

```diff
@@ -1,23 +1,65 @@
-import os
 import time
+import os
+import sys
+
+sys.path.append("..")
+import yaml
+import pickle
 from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from grafana import Grafana
```

Comment:
Can we make this import conditional on the fact the yaml configuration file describes the Grafana output?
There might be cases in which the grafana package is not installed but we do not need the output, so no reason to fail